src/components/SingleStoryCard/index.css

```diff
@@ -0,0 +1,46 @@
+.articleContainer {
+  margin: 3em auto 5em auto;
+  text-align: center;
+}
+
+.textBodyHolder {
+  width: 80%;
+  text-align: justify;
+  margin: auto;
+}
+
+.articleTitle {
+  color: #6c567b;
+  margin: 0rem 3rem 3rem 3rem;
+}
+
+.articleImage {
+  width: 35em;
+  margin: auto;
+}
+
+.articleBody {
+  line-height: 2rem;
+  margin: 3em auto;
+  color: #4a474d;
```

Comment:
Please, use variables for colors and rem instead of px

Reply:
changed